pom.xml

```diff
@@ -320,6 +320,11 @@
       <artifactId>grizzled-slf4j_${scala.version.prefix}</artifactId>
       <version>1.0.3</version>
     </dependency>
+
+    <!--
+      Only used directly in tests, but adding test-scope blocks transitive inclusion via Spark, resulting in runtime
```

Comment:
As a nice post-script to this discussion, the more recent ADAM snapshot we moved to in #581 depends on commons-io:2.4, so the exclusion is now unnecessary, and I've removed it.